python/cudf/cudf/tests/test_timedelta.py

```diff
@@ -528,7 +529,13 @@ def test_timedelta_series_mod_with_scalar_zero(reverse):
         datetime.timedelta(seconds=768),
         datetime.timedelta(microseconds=7),
         np.timedelta64(4, "s"),
-        np.timedelta64("nat", "s"),
+        pytest.param(
+            np.timedelta64("nat", "s"),
+            marks=pytest.mark.xfail(
+                strict=False,
```

Comment:
Can this not be made strict?

Reply:
Unfortunately no. This seems to work for some corner case combinations like empty inputs and same dtypes.